apps/bmd/src/App.tsx

```diff
@@ -33,16 +33,17 @@ export interface Props {
   screenReader?: ScreenReader
 }
 
-const App = ({
+const App: React.FC<Props> = ({
   screenReader = new AriaScreenReader(
     new SpeechSynthesisTextToSpeech(memoize(getUSEnglishVoice))
   ),
+
```

Comment:
weird that this only happened here but looks like an extra newline got randomly added here?

Reply:
I think I added that because of this: https://eslint.org/docs/rules/lines-between-class-members.